plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/wizard/AltConfigWizard.java

```diff
@@ -158,7 +158,23 @@ public IWizardPage getNextPage(final IWizardPage currentPage) {
 		}
 		return currentPage;
 	}
-
+	
+	public  String constructTemplateName() {

```

Comment:
If you implement your own method to assemble template names, please make sure to call it in the production code, too. There is little point in testing a method that is never used in the actual code.